### PR TITLE
UX-1193/Added ECO functionality and added an edit imported cluster sc…

### PR DESCRIPTION
**Pull request by kevinylee1991**
…reen

Screen Shot 2022-03-01 at 2 58 17 PM

No ECO installed:
Screen Shot 2022-03-01 at 12 45 16 PM
Screen Shot 2022-03-01 at 12 46 06 PM
Screen Shot 2022-03-01 at 1 13 48 PM

With ECO installed:
Screen Shot 2022-03-01 at 1 39 52 PM
Screen Shot 2022-03-01 at 1 48 00 PM
Screen Shot 2022-03-01 at 2 11 29 PM

### CODE REVIEW

**Comment by chrisj-platform9** [comment]
@kevinylee1991 the copy on the ECO tab info box should be as follows. 

-----
The External Cluster Operator is used to enable secure connectivity between external clusters and the Platform9 SaaS Management Plane. Download the YAML displayed below and run `Kubectl apply -f <path_to_yaml> `to install the ECO agent. 

Once installed the ECO Status will change to running.
-----

The copy on the General Tab should be: (I made some edits) 

-----
External Clusters require outbound connectivity from within the Cluster to the Platform9 SaaS Management Plane, this is achieved through the External Cluster Operator (ECO). ECO is responsible for creating a secure connection from the Cluster to Platform9. To manage this connection select the External Cluster Operator tab.
-----

**Comment by chrisj-platform9** [comment]
@kevinylee1991 please remove Monitoring from the Table Action Bar. This should be done via the the Add-on page.

File: src/app/plugins/kubernetes/components/infrastructure/importedClusters/ImportedClustersListPage.tsx [hunk 1]
```diff
@@ -10,8 +10,35 @@ import { isAdmin } from '../clusters/helpers'
...
 import SimpleLink from 'core/components/SimpleLink'
-import { importedClusterStatusMap } from './model'
+import { importedClusterStatusMap, IAgentStatus } from './model'
 import ClusterStatusSpan from '../clusters/ClusterStatus'
+import { IEditImportedClusterPageTabs } from './model'
+import FontAwesomeIcon from 'core/components/FontAwesomeIcon'
+import { makeStyles } from '@material-ui/styles'
+import Theme from 'core/themes/model'
+
+const getIconColor = (status: IAgentStatus, theme: Theme) =>
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/infrastructure/importedClusters/ImportedClustersListPage.tsx:20]: you shouldn't be referring to theme.palette anymore. We have a graph key with things like `error`, `unknown` and `success` to handle these specific colors

**Review by jruddell — APPROVED** [review]

File: src/app/plugins/kubernetes/components/infrastructure/importedClusters/ImportedClustersListPage.tsx (hunk 1 above)
Comment (kevinylee1991) [reply src/app/plugins/kubernetes/components/infrastructure/importedClusters/ImportedClustersListPage.tsx:20]: ah thanks! I missed the graph key when I was looking through the themes last time, that works perfectly

**Pull request merged**